tests/availability/processor.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=cdc.test
+
+MAX_RETRIES=20
+
+function test_processor_ha() {
+    test_stop_processor
+}
+
+# test_stop_processor stops the working processor
+# and then resume it.
+# We expect the data after resuming is replicated.
+function test_stop_processor() {
+    echo "run test case test_stop_processor"
+    # start a capture server
+    run_cdc_server $WORK_DIR $CDC_BINARY
+    # ensure the server become the owner
+    ensure $MAX_RETRIES "$CDC_BINARY cli capture list 2>&1 | grep '\"is-owner\": true'"
+    owner_pid=$(ps -C $CDC_BINARY -o pid= | awk '{print $1}')
+    owner_id=$($CDC_BINARY cli capture list 2>&1 | awk -F '"' '/id/{print $4}')
+    echo "owner pid:" $owner_pid
+    echo "owner id" $owner_id
+
+    # get the change feed id
+    changefeed=$($CDC_BINARY cli changefeed list 2>&1 | awk -F '"' '/id/{print $4}')
+    echo "changefeed id:" $changefeed
+
+    # stop the change feed job
+    # use ensure to wait for the change feed loading into memory from etcd
+    ensure $MAX_RETRIES "curl -s -d \"cf-id=$changefeed&admin-job=1\" http://127.0.0.1:8300/capture/owner/admin | grep true"
+
+    run_sql "INSERT INTO test.availability(id, val) VALUES (4, 4);"
+
+    # resume the change feed job
+    curl -d "cf-id=$changefeed&admin-job=2" http://127.0.0.1:8300/capture/owner/admin
```

Comment:
should we wait for the server is recovered and it has campaigned to be owner again. I found the test case fails sometimes

Reply:
The ownership was not changed ever, stopping a change feed does not resign an owner. Can you reproduce the fail case?

Reply:
It happens often, only find and no any processor is created.
```
[2020/04/07 14:50:33.589 +08:00] [INFO] [owner.go:365] ["handle admin job"] [changefeed=373f0adf-0bcf-421f-92d0-5c39fd0208a7] [type="stop changefeed"]
[2020/04/07 14:50:33.591 +08:00] [INFO] [etcd.go:200] ["show check lock"] [status="{\"table-infos\":[{\"id\":45,\"start-ts\":415823882176430095}],\"table-p-lock\":null,\"table-c-lock\":null,\"admin-job-type\":0}"]
```
But if sleep 1s before `resume job`, it always successes

Reply:
It happens often, only find and no any processor is created.
```
[2020/04/07 14:50:33.589 +08:00] [INFO] [owner.go:365] ["handle admin job"] [changefeed=373f0adf-0bcf-421f-92d0-5c39fd0208a7] [type="stop changefeed"]
[2020/04/07 14:50:33.591 +08:00] [INFO] [etcd.go:200] ["show check lock"] [status="{\"table-infos\":[{\"id\":45,\"start-ts\":415823882176430095}],\"table-p-lock\":null,\"table-c-lock\":null,\"admin-job-type\":0}"]
```
But if sleep 1s before `resume job`, it always successes